src/main/java/uk/gov/hmcts/reform/judicialapi/elinks/service/impl/IdamElasticSearchServiceImpl.java

```diff
@@ -196,6 +203,79 @@ public ResponseEntity<Object> getIdamElasticSearchSyncFeed() {
                 .body(judicialUsers);
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public ResponseEntity<Object> getIdamDetails() {
+        LocalDateTime schedulerStartTime = now();
+        elinkDataIngestionSchedularAudit.auditSchedulerStatus(JUDICIAL_REF_DATA_ELINKS,
+            schedulerStartTime,
+            null,
+            RefDataElinksConstants.JobStatus.IN_PROGRESS.getStatus(), IDAMSEARCH);
+        Set<IdamResponse> judicialUsers = new HashSet<>();
+        List<UserProfile> userProfiles = userProfileRepository.fetchObjectIdFromCurrentDate();
+        userProfiles.forEach(userProfile -> {
+            Map<String, String> params = new HashMap<>();
+            params.put("size",String.valueOf(recordsPerPage));
```

Comment:
do we need pagination for this? we are going to receive only one user record right?